app/models/user.rb

```diff
@@ -1,5 +1,8 @@
 class User < ApplicationRecord
-  validates :name, presence: true
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
- [ ] As we want (new) users to able to confirm their email kindly ensure that `confirmable` is enabled on device.

Reply:
Hello, @rloterh I added :confirmable to devise on line 4 but it is giving me an error after singing in `undefined local variable or method `confirmed_at' ` do you have any suggestion on how to solve it?

Reply:
Hi @EidHachem 
Sorry for the late response, I just noticed your message. However, I believe you've resolved this issue, as I can see from your this Pull Request.

Cheers!